transcendence-app/src/chat/chatroom/chatroom-member/infrastructure/db/postgres/chatroom-member.postgres.repository.ts

```diff
@@ -115,38 +116,49 @@ export class ChatroomMemberPostgresRepository
     userId: string,
   ): Promise<ChatroomMember | null> {
     const chatroomMembersData = await makeQuery(this.pool, {
-      text: `DELETE FROM ${this.table}
-      WHERE ${ChatroomMemberKeys.CHATID} = $1
-        AND ${ChatroomMemberKeys.USERID} = $2
-      RETURNING *;`,
+      text: `DELETE
+             FROM ${this.table}
+             WHERE ${ChatroomMemberKeys.CHATID} = $1
+               AND ${ChatroomMemberKeys.USERID} = $2
+             RETURNING *;`,
       values: [chatroomId, userId],
     });
     return chatroomMembersData && chatroomMembersData.length
       ? new this.ctor(chatroomMembersData[0])
       : null;
   }
 
-  async retrieveChatroomMembers(
+  async getPaginatedChatroomMembers(
     chatroomId: string,
+    paginationQueryDto: Required<PaginationWithSearchQueryDto>,
   ): Promise<ChatroomMemberWithUser[] | null> {
+    const { limit, offset, sort, search } = paginationQueryDto;
+
+    const orderBy =
```

Comment:
Do we plan to provide an option in frontend for choosing one way or another? If not maybe is just easier to sort by name

Reply:
What about during by joined at? Since we have it...